operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -339,4 +348,39 @@ public Counter getPeriodicReconciliationsCounter() {
     public AtomicInteger getResourceCounter() {
         return resourceCounter;
     }
+
+    /**
+     * Updates the resource state metric for the provided reconciliation which brings kind, name and namespace
+     * of the custom resource.
+     *
+     * @param reconciliation reconciliation to use to update the resource state metric
+     * @param state metric value
+     */
+    private void updateResourceState(Reconciliation reconciliation, int state) {
```

Comment:
I wonder if this would be more readable with `boolean ready` instead of `int state`. If nothing else, the Javadoc should list the supported states and explain them.

Reply:
Agree with the boolean. Just fixed that.